back/app/controllers/web_api/v1/invites_controller.rb

```diff
@@ -133,9 +133,8 @@ def accept
     invitee = @invite.invitee
     begin
       ActiveRecord::Base.transaction do
-        invitee.assign_attributes accept_params
+        UserService.assign_params_in_accept_invite(invitee, accept_params)
```

Comment:
I wonder whether it would be more explicit with all these calls to `UserService` to return and assign the user instead of changing it in place eg

```suggestion
        invitee = UserService.assign_params_in_accept_invite(invitee, accept_params)
```

Reply:
I agree. Done where it should not return boolean and makes sense in general.